help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+
+
+**Scheduling**
+1. Define the campaign schedule for your audience. <br>
+![Set the schedule](../../assets/catalog/data-partner/acxiom/scheduling.png)
+[!DNL Acxiom Audience Distribution] autogenerates a file name, so you do not have to configure it yourself. By default, the naming schema for file names is: `<Adobe Audience ID>_<DateTime Stamp>`.
+2. Once your schedule is defined, select **[!UICONTROL Create]** to save it.
+3. Select **[!UICONTROL Next]**.
+
+| Scheduling configuration  | Description                                |
+|---------------------------|-------------------------------------------------------------|
+| Export full files         | Select **Export full files**, as opposed to incremental files.<br> **Note**<br>[!DNL Acxiom’s Audience Connectors] does not currently support [Incremental file exports](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations).    |
+| Frequency                 | By default, the scheduling frequency is set to once and creates a one-time distribution. This option indicates how often the destination connection is invoked for distribution. Currently it is limited to one run. |
+| Scheduled                 | Select this option if you want to run your distributions at a scheduled time. Processing time is approximate.      |
+| Scheduled start time      | The timestamp for the projected run is presented in a UTC time zone. This tells [!DNL Acxiom] when to distribute the audience to the selected destination.  
+| After segment evaluation  | Select this option to have your distribution run right after it clears [!DNL Adobe’s] segment evaluation.    |
+| Date                      | Select the start date from the calendar display. Use the left and right carrot brackets (< and >) to move through the months of the calendar and select the date to set the start date.       |
+                                                          
+
+Once your audience is distributed to the selected destination, contact your destination representative to begin setting up your campaign.
+
+### Map attributes and identities {#map}
+For the [!DNL Acxiom Audience Distribution] destination to correctly receive the audience data, you must map the source fields from Experience Platform to the correct [!DNL Acxiom Audience Distribution] target fields.
+
+[!DNL Acxiom Audience Distribution] only allows mapping with the following fields and requires the profiles that come through our system to be in the following order: 
+
+| Field Name         | Description|Required| Field Order | Max Length |
+|--------------------|------------|--------|-------------|------------|          
+|First Name          |First name of individual                                                                                                         |N|1           |255         |

```

Comment:
For clarity, please use **Yes** and **No** in the **Required** column instead of **Y** and **N**.

Reply:
Done. Thank you!

Reply:
Fixed the last row. Thanks.